### PR TITLE
Fix: Some Dungeon Features to support changed Master Mode text changes in Artist's Adobe update

### PR DESCRIPTION
**Pull request by Fazfoxy**
## What
fixes Master Mode Stack Size

## Changelog Fixes
+ Fixed the Master Mode Dungeon stack size display in the party finder menu. - fazfoxy

### CODE REVIEW

**Comment by Thunderblade73** [comment]
Please add Regex Tests.
Example:
```Kotlin
/** REGEX-TEST: Abcs
*/
Repopattern.pattern("...","Abcs")
```

File: src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonFinderFeatures.kt [hunk 1]
```diff
@@ -57,9 +57,15 @@ object DungeonFinderFeatures {
         "note",
         "§7§7Note: §f(?<note>.*)"
     )
+    /* 
+    REGEX-TESTS:
```
Comment (Thunderblade73) [inline src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonFinderFeatures.kt]: The line REGEX-TESTS is not needed.

File: src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonFinderFeatures.kt [hunk 2]
```diff
@@ -57,9 +57,15 @@ object DungeonFinderFeatures {
...
+    * REGEX-TEST: The Catacombs
+    * REGEX-TEST: Master Mode The Catacombs
+    * REGEX-TEST: MM Catacombs
```
Comment (Thunderblade73) [inline src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonFinderFeatures.kt]: This test can't match that better. So either the pattern is wrong or the test is wrong

**Review by hannibal002 — CHANGES_REQUESTED** [review]
The master star items still have no stack number.
grafik


or was this pr fixing somehting else?

**Comment by Fazfoxy** [comment]
image
This isn't what I'm fixing but also, those stack sizes are working locally, this feature does sometimes fail to work though maybe re-check it?

**Comment by hannibal002** [comment]
what else are you fixing then?

**Comment by Thunderblade73** [comment]
@hannibal002 I think your "repo pattern force locale" is on the wrong value. If you are in dev enviroment the option must be off. At least that would be the obvious case why it wouldn't work.

**Comment by Fazfoxy** [comment]
more specifically, this was to do with hypixel's changing of PF notes to be Master Mode The Catacombs unlike previously + dungeon clean end being updated for changed lines

**Comment by hannibal002** [comment]
ok i still dont know what the fix actually is about :/ @Fazfoxy

**Comment by Fazfoxy** [comment]
> ok i still dont know what the fix actually is about :/ @Fazfoxy

ok so, it does fix the MM highlight apparently, but also it makes party stack sizes go back to the correct floor, before PR MM parties would stacksize as the non MM variant because of hypixel changing the description + should fix clean end because hypixel changed that line as well I believe

**Comment by Fazfoxy** [comment]
image
when using non changed it should show like this for MM floors

**Comment by hannibal002** [comment]
Now I see, thanks. I have changed the changelog wording accordingly.

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**